help/destinations/catalog/email-marketing/salesforce-marketing-cloud-exact-target.md

```diff
@@ -0,0 +1,213 @@
+---
+keywords: email;Email;e-mail;email destinations;salesforce;salesforce destination
+title: Salesforce Marketing Cloud (ExactTarget) connection
+description: The Salesforce Marketing Cloud (ExactTarget) destination allows you to export your account data and activate it within Salesforce Marketing Cloud (ExactTarget) for your business needs.attribute set
```

Comment:
The Salesforce Marketing Cloud (formerly known as ExactTarget) destination allows you to [...]